aries-site/src/components/cards/ContentCard.js

```diff
@@ -6,15 +6,15 @@ import { Identifier, Tile } from 'aries-core';
 import { PreviewImageCard } from './PreviewCard';
 import { useDarkMode } from '../../utils';
 
-const StyledTile = styled(Tile)`
+export const StyledTile = styled(Tile)`
   transition: all 0.3s ease-in-out;
   :focus,
   :hover {
     transform: scale(1.01, 1.01);
   }
 `;
 
-export const ContentCard = forwardRef(({ topic, ...rest }, ref) => {
+export const ContentCard = forwardRef(({ topic, minimal, ...rest }, ref) => {
```

Comment:
Doesn't need to be adjusted for this PR, but "minimal" has got me thinking about something along the lines of t-shirt sizing cards, where card content sections show/hide based on t-shirt size, similar to `minimal` here. Probably more of a topic for discussion as Card elaboration is considered.

Reply:
Would be an interesting concept to discuss for sure.